stories/button/button.stories.js

```diff
@@ -41,40 +41,38 @@ export const primary = () => `
  */
 
 export const types = () => `<div class="fddocs-container fddocs-button-container"> 
-        <button class="fd-button">Default Button</button>
-        <button class="fd-button fd-button--emphasized">Emphasized Button</button>
-        <button class="fd-button fd-button--ghost">Ghost Button</button>
+        <button class="fd-button fd-button--emphasized">Primary Button</button>
```

Comment:
I think we need to explain the mapping between the 2 types - one is the use of the buttons and the other is its design. not just to replace the current.
Maybe in another example document which of the designs are primary, secondary, tertiary buttons